jython/LifxLightControlExample.py

```diff
@@ -0,0 +1,46 @@
+###################################################################################################
+# Demonstration of controling LIFX strip lights.
```

Comment:
No need to change, but a suggestion is to add a link to LIFX. It makes it easier to understand what it's about.
https://www.lifx.com/